src/Sulu/Bundle/ContactBundle/Controller/AccountController.php

```diff
@@ -213,9 +140,11 @@ public function getContactsAction($id, Request $request)
             $this->restHelper->initializeListBuilder($listBuilder, $fieldDescriptors);
 
             $listBuilder->addSelectField($fieldDescriptors['contactId']);
-            $listBuilder->setIdField($fieldDescriptors['id']);
-            $listBuilder->where($fieldDescriptors['account'], $id);
-            $listBuilder->sort($fieldDescriptors['lastName'], $listBuilder::SORTORDER_ASC);
+            if ($listBuilder instanceof DoctrineListBuilder) {
```

Comment:
We should avoid such changes in this kind of refactors. Also in this case we should just assert that `$listBuilder` is `DoctrineListBuilder` so `\assert($listBuilder instanceof DoctrineListBuilder);` after `$this->listBuilderFactory->create` would be better. but also `@var` is in this case enough as the `DoctrineListBuilderFactoryInterface` always returns a `DoctrineListBuilder` and the controller should so not care and would require to fix it with generics in the list builder interfaces classes itself. But as said that kind of tasks should be in seperate pull request to keep things concentrated on one single refactors.